core/src/filed/backup.h

```diff
@@ -72,6 +72,10 @@ void StripPath(FindFilesPacket* ff_pkt);
 void UnstripPath(FindFilesPacket* ff_pkt);
 int SaveFile(JobControlRecord* jcr, FindFilesPacket* ff_pkt, bool top_level);
 
+// allow to set the allowed maximum to something lower for testing purposes
+#ifndef MAXIMUM_ALLOWED_FILES_PER_JOB
+#define MAXIMUM_ALLOWED_FILES_PER_JOB std::numeric_limits<decltype(JobControlRecord::JobFiles)>::max()
+#endif
```

Comment:
This is fine for now, but we should think of a way to do this in which it is not only testable but also _tested_.

Maybe for 25 we can change this to be a run time value that you can set when starting the FD.  This way we can actually set up an automated test for it!